Jenkinsfile

```diff
@@ -234,12 +234,53 @@ timestamps {
 
       try {
         stage("Build docker environment") {
-          sh("make build")
+          withEnv([
+            "ASSET_MANAGER_COMMITISH=${params.ASSET_MANAGER_COMMITISH}",
+            "CONTENT_STORE_COMMITISH=${params.CONTENT_STORE_COMMITISH}",
+            "GOVERNMENT_FRONTEND_COMMITISH=${params.GOVERNMENT_FRONTEND_COMMITISH}",
+            "PUBLISHING_API_COMMITISH=${params.PUBLISHING_API_COMMITISH}",
+            "ROUTER_API_COMMITISH=${params.ROUTER_API_COMMITISH}",
+            "RUMMAGER_COMMITISH=${params.RUMMAGER_COMMITISH}",
+            "SPECIALIST_PUBLISHER_COMMITISH=${params.SPECIALIST_PUBLISHER_COMMITISH}",
+            "STATIC_COMMITISH=${params.STATIC_COMMITISH}",
+            "TRAVEL_ADVICE_PUBLISHER_COMMITISH=${params.TRAVEL_ADVICE_PUBLISHER_COMMITISH}",
+            "COLLECTIONS_PUBLISHER_COMMITISH=${params.COLLECTIONS_PUBLISHER_COMMITISH}",
+            "COLLECTIONS_COMMITISH=${params.COLLECTIONS_COMMITISH}",
+            "PUBLISHER_COMMITISH=${params.PUBLISHER_COMMITISH}",
+            "FRONTEND_COMMITISH=${params.FRONTEND_COMMITISH}",
+            "CALENDARS_COMMITISH=${params.CALENDARS_COMMITISH}",
+            "MANUALS_PUBLISHER_COMMITISH=${params.MANUALS_PUBLISHER_COMMITISH}",
+            "MANUALS_FRONTEND_COMMITISH=${params.MANUALS_FRONTEND_COMMITISH}",
+            "WHITEHALL_COMMITISH=${params.WHITEHALL_COMMITISH}",
+          ]) {
```

Comment:
Rather than having 3 withenv blocks shall we just set the environment variables at the start of the jenkinsfile?

I think we could make use of the [setEnv in govuk groovy](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy#L153-L158) or use an [environment block](https://jenkins.io/doc/pipeline/tour/environment/)